tests/integration/workflows/test_utils.py

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of INSPIRE.
+# Copyright (C) 2014-2017 CERN.
+#
+# INSPIRE is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# INSPIRE is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE. If not, see <http://www.gnu.org/licenses/>.
+#
+# In applying this license, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization
+# or submit itself to any jurisdiction.
+
+"""Tests for Workflows utils."""
+
+from __future__ import absolute_import, division, print_function
+
+import pytest
+
+from invenio_db import db
+
+from inspirehep.modules.workflows.utils import store_original_record_for_source, retrieve_original_record_for_source
+
+
+@pytest.fixture()
+def dummy_record(small_app):
+    from invenio_records.api import Record
```

Comment:
> Why is this import internal?

Was thinking to import globally the functions that are under tests, while import locally the rest of the auxiliary functions. But indeed no other good reason.

> because its `delete` plays nicer with the tests.

Mmh. `InspireRecord.delete()` is not good for tests because it's a soft delete, so tables are not well cleaned afterwards. On the other hand I see can user `InspireRecord._delete()` which is implementing Invenio's one, which rightfully can implement a `force`-delete, and apply all the ondelete-cascade triggers.